.gitignore

```
@@ -10,3 +10,6 @@ luacov.report.out
 luacov.stats.out
 build/*.cmake
 build/Makefile
+
+# Vim Swap files.
+.*.sw[a-z]
```

http://vimdoc.sourceforge.net/htmldoc/recover.html#swap-file
```
If the ".swp" file name already exists, the last character is
	   decremented until there is no file with that name or ".saa" is
	   reached.  In the last [case](http://vimdoc.sourceforge.net/htmldoc/change.html#case), no swap file is created.
```
```suggestion
.*.s[a-w][a-z]
```

---

No problem, updated.